schema_editor/test/spec/schemas/schemas-service.spec.js

```diff
@@ -148,4 +176,69 @@ describe('ase.schemas:Schemas', function () {
             message: jasmine.stringMatching('Diff')
         }]));
     });
+
+    // NOTE: This test must be updated when new field types are added.
+    it('should preserve all information when serializing and deserializing schemas', function () {
+        // In practice this means that the serialize and deserialize methods are inverses
```

Comment:
I was trying to think of a way to do that, but ran into the same problem identified here: https://github.com/WorldBank-Transport/DRIVER/pull/31#commitcomment-11407950 , which is that it's not easy right now to test things that are in `builder-schemas` because they're in separate files. I'd definitely like to add this test if we can find a way to do it.